### PR TITLE
add some qol

### PR DESCRIPTION
**Pull request by float3**

### CODE REVIEW

File: Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs [hunk 1]
```diff
@@ -102,6 +114,8 @@ private static void ApplyToCurrentAvatar()
                 BuildReport.Clear();
 
                 ProcessAvatar(avatar);
+                avatar.SetActive(false);
```
Comment (bdunderscore) [inline Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs:117]: Hmm. Disabling the original avatar is good for some workflows, but gets in the way in others (in particular I'll often bake and rebake the same avatar a lot of times while debugging 😅 ). I think I'd prefer to leave this step out?

Comment (float3) [reply Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs:117]: Does disabling prevent that though? you can bake a disabled Avatar iirc. if you still don't want this I'll remove that step.

Comment (bdunderscore) [reply Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs]: I guess I'm wondering why disabling is needed; we already shift the position of the avatar so you can visually check that things look right.

Comment (float3) [reply Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs]: https://feedback.vrchat.com/sdk-bug-reports/p/sdk-disables-all-publish-buttons-if-just-one-avatar-in-scene-has-an-error

Comment (float3) [reply Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs]: While MA's components might cause this other tools' components do

Comment (float3) [reply Packages/nadena.dev.modular-avatar/Editor/AvatarProcessor.cs]: I wonder why MA's components don't cause this bug?

**Pull request merged**